### PR TITLE
Update nansat_wkv.yml

### PR DESCRIPTION
**Pull request by opsdep**

### CODE REVIEW

File: nansat_wkv.yml [hunk 1]
```diff
@@ -491,3 +491,18 @@
   units: celsius
   minmax: -2 40
   colormap: jet
+
+- standard_name: y_sea_water_velocity
+  long_name: Y Sea Water Velocity
+  short_name: U
```
Comment (akorosov) [inline nansat_wkv.yml]:
```suggestion
  short_name: V
  units: m s-1
```

File: nansat_wkv.yml [hunk 2]
```diff
@@ -491,3 +491,18 @@
...
+  units: m s-1
+  minmax: -20 20
+  colormap: jet
+  
+- standard_name: x_sea_water_velocity
+  long_name: X Sea Water Velocity
+  short_name: V
```
Comment (akorosov) [inline nansat_wkv.yml]:
```suggestion
  short_name: U
  units: m s-1
```

**Review by akorosov — CHANGES_REQUESTED** [review]

**Comment by opsdep** [comment]
@akorosov I have committed them.

File: nansat_wkv.yml [hunk 3]
```diff
@@ -491,3 +491,20 @@
   units: celsius
   minmax: -2 40
   colormap: jet
+
+- standard_name: y_sea_water_velocity
+  long_name: Y Sea Water Velocity
+  short_name: V
+  units: m s-1
+  units: m s-1
```
Comment (akorosov) [inline nansat_wkv.yml]: Oops, now the units are duplicated. Both here and below. I accidentally included them in the suggestion.
```suggestion
```

File: nansat_wkv.yml [hunk 4]
```diff
@@ -491,3 +491,20 @@
...
+  minmax: -20 20
+  colormap: jet
+  
+- standard_name: x_sea_water_velocity
+  long_name: X Sea Water Velocity
+  short_name: U
+  units: m s-1
```
Comment (akorosov) [inline nansat_wkv.yml:505]:
```suggestion
```

**Review by akorosov — CHANGES_REQUESTED** [review]

**Review by akorosov — APPROVED** [review]

**Pull request merged**